test/verifiedTests/analysis/testMultiSpeciesModelling/testTranslateMetagenome2AGORA.m

```diff
@@ -0,0 +1,55 @@
+% The COBRAToolbox: translateMetagenome2AGORAn.m
+%
+% Purpose:
+%     - tests the basic functionality of translateMetagenome2AGORA
+%
+% Authors:
+%     - Almut Heinken - March 2019
+%
+% global CBTDIR
```

Comment:
Why are all these lines commented out?

Reply:
Sorry, that was by mistake, I meant to delete these lines later. I will address your comments.